pkg/server/backuprestoreserver.go

```diff
@@ -269,7 +261,7 @@ func (b *BackupRestoreServer) runEtcdProbeLoopWithSnapshotter(ctx context.Contex
 		const recentFullSnapshotPeriodInHours = 23.5
 		initialDeltaSnapshotTaken = false
 		if ssr.PrevFullSnapshot != nil && !ssr.PrevFullSnapshot.IsFinal && time.Since(ssr.PrevFullSnapshot.CreatedOn).Hours() <= recentFullSnapshotPeriodInHours {
-			ssrStopped, err := ssr.CollectEventsSincePrevSnapshot(ssrStopCh)
+			ssrStopped, err := ssr.CollectEventsSincePrevSnapshot(ctx)
```

Comment:
Is there any specific reasons to replace the `ssrStopCh` with `ctx`?

Reply:
See https://github.com/gardener/etcd-backup-restore/pull/406#discussion_r754315829.